packages/appstore/src/components/add-more-wallets/add-more-wallets.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text, Loading } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useAvailableWallets } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import CarouselContainer from './carousel-container';
+import AddWalletCard from './wallet-add-card';
+
+import './add-more-wallets.scss';
+
+const AddMoreWallets = observer(() => {
+    const { data, isLoading } = useAvailableWallets();
+
+    if (!data) return <Loading is_fullscreen={false} />;
```

Comment:
```suggestion
```
`data` would not be available if it is loading or the request is not resolved. The check below should be sufficient

Reply:
@yashim-deriv Yes and we want to show the Loading component when the data is not available.
Which check you mean?

Reply:
i think this is referring to the loading component below in the CarouselContainer, it should be enough to have the below Loading component since we already have isLoading check below 👍

Reply:
@mahdiyeh-deriv just to confirm again, do we need the above Loading since we already have loading below?